hazelcast/src/main/java/com/hazelcast/internal/serialization/SerializationService.java

```diff
@@ -38,6 +38,8 @@
      */
     <B extends Data> B toData(Object obj);
 
+    <B extends Data> B toDataWithSchema(Object object);
```

Comment:
Add javadoc, same for `trimSchema` below

Reply:
Javadocs are on their way.

Reply:
Fixed by @mdumandag 539aaf7